.test-infra/jenkins/job_PostCommit_Java_PortableValidatesRunner_Flink.groovy

```diff
@@ -36,7 +36,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_PVR_Flink',
   steps {
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks(':beam-runners-flink_2.11-job-server:validatesPortableRunner')
+      tasks(':beam-runners-flink_2.11-job-server:flinkCompatibilityMatrixDocker')
```

Comment:
This replaces the Java PVR tests with the Python validation tests. Was this intended?

Reply:
oh, actually that's just an old name for the same target that I used while developing (which I did copy from the python `build.gradle`, you're right)

I think as is, this would just fail because there's no target with that name in `…/job-server/build.gradle`, right?

I am missing where the configuration is that maps the "Run Java Flink PortableValidatesRunner" command to the target `:beam-runners-flink_2.11-job-server:validatesPortableRunner`, can you point me to it?

Thanks! I will change it back to `validatesPortableRunner` in the meantime, as well.

Reply:
Yes, this should fail since the task is in the python sdk module.

>I am missing where the configuration is that maps the "Run Java Flink PortableValidatesRunner" command to the target :beam-runners-flink_2.11-job-server:validatesPortableRunner, can you point me to it?

It's here:
https://github.com/apache/beam/blob/93810316dfbe43894e8fb8da6e10a6180ea536d2/runners/flink/job-server/build.gradle#L75

Reply:
Ah, I suppose you mean the Jenkins task, that one is in `.test-infra/jenkins/job_PostCommit_Java_PortableValidatesRunner_Flink.groovy`.